src/Components/ClustersListTable/ClustersListTable.spec.ct.js

```diff
@@ -7,7 +7,7 @@ import _ from 'lodash';
 import { Intl } from '../../Utilities/intlHelper';
 import getStore from '../../Store';
 import { ClustersListTable } from './ClustersListTable';
-import props from '../../../cypress/fixtures/ClustersListTable/data.json';
+import props from '../../../cypress/fixtures/api/insights-results-aggregator/v2/clusters/report.json';
```

Comment:
maybe just
```suggestion
import props from '../../../cypress/fixtures/api/insights-results-aggregator/v2/clusters.json';
```
?

https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/server/api/v2/openapi.json#L424

Reply:
fine with me, I just wanted to follow the approach that was there. I think this change can be applied to all endpoints. Do you agree to change it everywhere?

Reply:
updated, please take a look